lib/logstash/environment.rb

```diff
@@ -76,10 +76,6 @@ def jruby?
       @jruby ||= !!(RUBY_PLATFORM == "java")
     end
 
-    def windows?
-      Gem.win_platform?
-    end
```

Comment:
Need to be used in the plugins manager to set defaults for packaging and unpack.